theme/components/button.scss

```diff
@@ -71,13 +46,21 @@
 
 .ck-button,
 a.ck-button {
-	display: inline-block;
+	@include ck-button-colors( 'background' );
+	@include ck-rounded-corners();
+
+	border-width: 1px;
+	border-style: solid;
+	white-space: nowrap;
+	cursor: default;
+	vertical-align: middle;
 	padding: ck-spacing( 'small' );
+	line-height: $ck-icon-size / $ck-font-size-base;
 
-	@include ck-button( 'background' );
-	@include ck-unselectable();
-	@include ck-rounded-corners();
-	@include ck-tooltip_enabled();
+	&:focus {
+		@include ck-focus-ring();
+		@include ck-box-shadow( $ck-focus-outer-shadow );
+	}
 
 	@include ck-button-icon {
 		float: left;
```

Comment:
+L Maybe simply set ```display: inline-block``` for buttons in wireframe theme. Float left seems to be more problematic in connection with another element without float value.